superset-frontend/packages/generator-superset/generators/plugin-chart/templates/README.erb

```diff
@@ -1,52 +1,40 @@
-## superset-plugin-chart-<%= packageName %>
+# <%= packageName %>
 
-This plugin provides <%= description %> for Superset.
+This is the <%= description %>.
 
 ### Usage
 
-Configure `key`, which can be any `string`, and register the plugin. This `key` will be used to lookup this chart throughout the app.
+To build the plugin, run the following commands:
 
-```js
-import <%= packageLabel %>ChartPlugin from '@superset-ui/plugin-chart-<%= packageName %>';
+```
+npm i --force
+npm run build
+```
+
+To run the plugin in development mode (=rebuilding whenever changes are made), start the dev server with the following command:
 
-new <%= packageLabel %>ChartPlugin()
-  .configure({ key: '<%= packageName %>' })
-  .register();
+```
+npm run dev
+```
+
+To add the package to Superset, go to the `superset-frontend` subdirectory in your Superset source folder (assuming both the `superset-plugin-chart-liquid` plugin and `superset` repos are in the same root directory) and run
```

Comment:
Hey @villebro I am not sure what `superset-plugin-chart-liquid` is referring to. Is this just a sample name?

Reply:
thanks for catching! It's an example plugin that I've worked on while testing this template